packages/hash/integration/src/tests/model/link.model.test.ts

```diff
@@ -299,6 +299,136 @@ describe("Link model class ", () => {
     );
     expect(entityATimestamp4OutgoingLinks[0]!.index).toBe(0);
   });
+
+  it("update method correctly updates link with a non-versioned source entity", async () => {
+    const { accountId } = existingUser;
+
+    const stringifiedPath = "$.test";
+
+    const entityA = await Entity.createEntityWithLinks(db, {
+      user: existingUser,
+      accountId,
+      entityDefinition: {
+        versioned: true,
```

Comment:
Test specifies `correctly updates link with a non-versioned source entity` but this seems to be versioned?

Or am I mistaken and this is the destination entity?

Reply:
Oops, yes I've implemented them the wrong way around. Fixed in https://github.com/hashintel/hash/pull/403/commits/58b40c691c43cfec6a8b0d268d36d8621aac6729